formats/README.md

```diff
@@ -16,6 +16,20 @@ Allows deserialization of `Config` object from popular [lightbend/config](https:
 into Kotlin objects.
 You can learn about "Human-Optimized Config Object Notation" or HOCON from library's [readme](https://github.com/lightbend/config#using-hocon-the-json-superset).
 
+## ProtoBuf
+
+* Artifact id: `kotlinx-serialization-protobuf`
```

Comment:
btw, it's not clear that these artifacts have the same naming conventions (e.g. `kotlinx-serialization-protobuf-js` for js, `-native` for native etc). Better to highlight this somewhere (maybe in `Platform:` section?)

Reply:
I am not sure it is worth documenting MPP usages in every library.

Reply:
It's about discoverability

Reply:
Let's do it properly in a separate PR then: add `README` to every platform-specific source set with a way to add the dependency, and mention that in the root readme with a proper references

Reply:
Yeah, okay. Maybe we'll come up with a better documentation structure (or eventually publish metadata for common artifact)